addons/account/views/account_move_views.xml

```diff
@@ -962,29 +962,51 @@
                                 <label for="journal_id"
                                        groups="account.group_account_readonly"
                                        invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
+                                <label for="currency_id"
+                                       groups="!account.group_account_readonly,base.group_multi_currency"
```

Comment:
Isn't it a `or`? Don't you want a `and`? (iirc, but I haven't tested)

Reply:
It should work as intended since an `!` is involved It roughly works as `not any(<groups preceded by !>) and any(<groups not preceded by !>)`  [^1].
I basically just took it from the `currency_id` field for community that I deleted.
https://github.com/odoo/odoo/pull/147910/files#diff-a13b3dfd8d9e1c45eee3b202345ad950f6b90db975708ee34b9bff2886f4513dL985
So since it worked before it should still work.

[^1]: https://github.com/odoo/odoo/blob/7369cc291f5f30b8b312d6c1f9dada3fe83e06b3/odoo/models.py#L1557